newdle/client/src/components/Home.js

```diff
@@ -1,15 +1,74 @@
 import React from 'react';
-import {useSelector} from 'react-redux';
-import {Link} from 'react-router-dom';
-import {isLoggedIn} from '../selectors';
+import {Button} from 'semantic-ui-react';
+import LoginRequired from './LoginRequired';
+import {useRouter} from '../util/router';
+import {Icon} from 'semantic-ui-react';
+
+import {ReactComponent as TeamIcon} from '../images/team.svg';
+import {ReactComponent as TimetableIcon} from '../images/timetable.svg';
+import {ReactComponent as DiscussIcon} from '../images/discuss-issue.svg';
+import {ReactComponent as GitHubIcon} from '../images/github.svg';
+
+import styles from './Home.module.scss';
 
 export default function Home() {
-  const isUserLoggedIn = useSelector(isLoggedIn);
+  const router = useRouter();
 
   return (
-    <>
-      <h2>Home</h2>
-      {isUserLoggedIn && <Link to="/new">Create newdle</Link>}
-    </>
+    <div>
+      <div className={styles.container}>
+        <h2>How does it work?</h2>
+        <ol className={styles.box}>
+          <li>
+            <TeamIcon className={styles.icon} />
+            Choose your participants
+          </li>
+          <li>
+            <TimetableIcon className={styles.icon} />
+            Set the time slots based on their availability
+          </li>
+          <li>
+            <DiscussIcon className={styles.icon} />
+            <strong>newdle</strong> will collect the answers!
```

Comment:
It looks strange when other list items start with capital letters and this one doesn't. Maybe capital `N` in newdle in this case?

Reply:
I was undecided, because the newdle logo is lowercase...